subsys/bootloader/Kconfig

```diff
@@ -117,6 +117,12 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
+config SB_DEBUG_NO_VERIFY_HASHES
+	bool "[DEBUG] Don't check public key hashes for applicability"
+	default n
```

Comment:
We typically don't do `default n` since its the default :smile:

Reply:
Done